packages/autorun-luajit/src/types/common.rs

```diff
@@ -0,0 +1,427 @@
+// Subset of lj_obj.h
+
+// IMPORTANT: GMod's LUA_IDSIZE was randomly changed to 128 instead of 60 like in vanilla LuaJIT
+pub const LUA_IDSIZE: i32 = 128;
```

Comment:
I know this package probably won't ever be promised to be used outside of gmod, but it would be nice to gate this under a feature "gmod" to make it explicit